src/tests/ngraph_helpers/snippets_ngraph_functions/include/precision_propagation_function.hpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <ngraph/ngraph.hpp>
+#include "ngraph/opsets/opset1.hpp"
+#include "snippets/op/convert_saturation.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+class PrecisionPropagationFunction {
```

Comment:
I suggest to keep it simple as possible. The type has one method only to test separate standalone transformartion. It's so simple and doesn't need to have any interface. If not, can you specify: why we need additional interface?

Anyway, there is separate commit with `SnippetsFunctionBase` implementation: `smoke_Snippets_PrecisionPropagationTest: empty stub was added` (will be changed after rebase: https://github.com/openvinotoolkit/openvino/pull/14996/commits/6ccf2ba5c8bc7bb4a4448890eb0ea0b49c71f1c0). I hope it will be  rolled back after ofline discussion.